### PR TITLE
Dashboard Home Tab

### PR DESCRIPTION
**Pull request by ChazUK**
Changed contact phone number on Contact Page
Changed email on Dashboard
Made the Service name a link to the Dashboard
Updated the Home Tab to include links and notifications
Next Step alerts now display on a conditional
Added NMDS ID and Last Signed In Date to Dashboard
Truncating fullname to first name

### CODE REVIEW

**Comment by ChazUK** [comment]
[Account home page/ User Dashboard](https://trello.com/c/kXN5haL1/67-account-home-page-user-dashboard)

**Comment by ChazUK** [comment]
[Change Request - Contact Us Details](https://trello.com/c/3jKnPtwd/133-change-request-contact-us-details)

**Comment by ChazUK** [comment]
[NMDS-ID to become Workplace ID - to be displayed on Establishments](https://trello.com/c/Ept21xD8/70-nmds-id-to-become-workplace-id-to-be-displayed-on-establishments)

**Comment by ChazUK** [comment]
[Home Page - Header review](https://trello.com/c/9w5BwDzC/129-home-page-header-review)

File: src/app/core/services/auth-service.ts [hunk 1]
```diff
@@ -27,6 +27,7 @@ interface LoggedInEstablishment {
 interface LoggedInSession {
   fullname: string;
   isFirstLogin: boolean;
+  lastLogin: string;
```
Comment (wozzer72) [inline src/app/core/services/auth-service.ts]: The login response returns `lastLoggedIn` - `server/routes/login.js`

File: src/app/features/dashboard/dashboard.component.html [hunk 2]
```diff
@@ -1,36 +1,34 @@
...
+
+  <div class="row">
+    <div class="col-md-12">
+      <h2 class="h2 fs-36 mb-4">{{ establishment?.name }}</h2>
+    </div>
+    <div class="col-md-6">
+      <p>Workplace ID: {{ establishment?.nmdsId }}</p>
+    </div>
+    <div class="col-md-6">
+      <p class="text-right" *ngIf="lastLogin">Last signed in {{ lastLogin | date : 'd MMMM y' }}</p>
```
Comment (wozzer72) [inline src/app/features/dashboard/dashboard.component.html]: The login response field name is `lastLoggedIn`.

File: src/app/core/services/auth-service.ts (hunk 1 above)
Comment (ChazUK) [reply src/app/core/services/auth-service.ts]: Updated

**Pull request merged**